src/runtime/node/perf_hooks/internal/performance.ts

```diff
@@ -44,6 +67,22 @@ export const Performance = class Performance
     return entry as any;
   }
 
+  markResourceTiming(
+    timingInfo: object,
+    requestedUrl: string,
+    initiatorType: string,
+    global: object,
+    cacheMode: string,
+    bodyInfo: object,
+    responseStatus: number,
+    deliveryType?: string,
+  ): PerformanceResourceTiming {
+    // TODO: create a new PerformanceResourceTiming entry
+    // so that peformance.getEntries, getEntriesByName, and getEntriesByType return it
+    // see: https://nodejs.org/api/perf_hooks.html#performancemarkresourcetimingtiminginfo-requestedurl-initiatortype-global-cachemode-bodyinfo-responsestatus-deliverytype
+    return mock;
```

Comment:
We have `_PerformanceResourceTiming` class perhaps we can return an instance of it?

Reply:
addressed in c5345ad8c914fec4a55ddab39e15d605dbfb7480